### DISCUSSION TITLE
Getting selective alerts

### DISCUSSION THREAD

**Discussion by krkeshav** [SigNoz/signoz#5421] [Q&A]
Hi, 
I raised this question in recent community call.
We have a critical external API call linked to multiple microservices in a linear chain. If the response time of this API is very high, can we set up alerts specifically for this API instead of getting alerts for all dependent layers? 

We need this as this creates lots of alerts on each layer which creates noise. At the same time if we have an actual response time increase issue on our service instead of external API then we want that alert for that service which caused that issue.

**Comment by Calm-Rock** [comment]
@makeavish @srikanthccv pls have a look at this.
